packages/grantee-sdk-js/src/idOS-grantee.ts

```diff
@@ -125,17 +118,15 @@ export class idOSGrantee {
     dbId?: string;
     chainType: "NEAR";
     granteeSigner: KeyPair;
-    granteeOptions?: NearGrantsOptions;
   }): Promise<idOSGrantee>;
 
   static async init({
     recipientEncryptionPrivateKey,
     nodeUrl = KwilWrapper.defaults.kwilProvider,
-    chainId,
+    chainId = KwilWrapper.defaults.chainId,
```

Comment:
Why are we adding this default? This will preclude discovery from being the default behavior, which feels wrong.